cob_hardware_config/cob3-2/config/voltage_filter.yaml

```diff
@@ -0,0 +1,8 @@
+abcd:
+- 4.445526107917127e-08
+- -0.005832227951854395
+- 257.41863195497723
+- -3821968.813311375
+maximum_time: 11166.85982958223
+off_y: 6000
+theta: -0.14999999999999997
```

Comment:
this file is not needed anymore as it now is called battery_voltage_filter.yaml, right?